.github/workflows/release.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         submodules: true
     - name: Configure
       run: |
-          apt update && apt install -y debhelper moreutils
+          apt update && apt install -y debhelper moreutils pip jq
```

Comment:
we have several "apt install" calls in our actions. Let's eliminate them by adding all these packages to `ghcr.io/romange/ubuntu-dev` .  Feel free to send a PR to https://github.com/romange/container-foundry

Reply:
I already opened a PR. 

1. I would like to add regression tests for arm as well. I know I said offline that it doesn't worth our time, but I am already familiar with most of the gh action staff now that I should be able to find a good solution (and my OCD won't just let me to only run it for x86). The issue is that I could not run the regression tests action as part of the `run-on-aarch` action. I could simply not find a way to do this properly. Moreover, we only have x86 runners so running on a pure aarch machine was a big no and therefore we require QEMU. But as I looked on your repo, it seems that you already using QEMU which means that I could reuse your images instead of `run-on-aarch` and therefore I would be able to run that action as separate step (because the OG issue, that I can't run an action in the context of another action, that is, the `run-on-aarch`).
2. As I said I made this PR, but I can clean up the release even more once I get the regression tests running on a container explicitly (as opposed to the action). I won't wait for my PR to be merged. I will open a new one to clean up everything.